1.	Identify-Basic-Internet-Problems-with-ping.md

```diff
@@ -0,0 +1,113 @@
+
+```
+MacBook-Pro:~ ajm$ ping google.com
+[Note: Queue crickets, wait a while...]
+ping: cannot resolve google.com: Unknown host
+MacBook-Pro:~ ajm$
+```
+
+Or, for example #2, depending on exactly how the connection is failing:
+```
+PING google.com (216.58.196.46): 56 data bytes
+Request timeout for icmp\_seq 0
+Request timeout for icmp\_seq 1
+Request timeout for icmp\_seq 2
+^C
+```
+
+And sometimes, if I have a particularly flaky connection, I'll see a mixture of these messages. For example #3, I can simulate this by connecting my Mac computer to a public Wi-Fi connection that is across the street:
+```
+PING google.com (216.58.196.206): 56 data bytes
+64 bytes from 216.58.196.206: icmp\_seq=0 ttl=57 time=273.655 ms
+64 bytes from 216.58.196.206: icmp\_seq=1 ttl=57 time=808.546 ms
+64 bytes from 216.58.196.206: icmp\_seq=2 ttl=57 time=179.613 ms
+Request timeout for icmp\_seq 3
+Request timeout for icmp\_seq 4
+64 bytes from 216.58.196.206: icmp\_seq=5 ttl=57 time=374.612 ms
+Request timeout for icmp\_seq 6
+ping: sendto: No route to host
+Request timeout for icmp\_seq 7
+ping: sendto: No route to host
+Request timeout for icmp\_seq 8
+^C
+```
+
+In the first test, "ping" told me that my machine could not even find the Internet address (IP 216.58.196.46) for Google.com. In the second test, my computer remembered Google's IP address, but could not actually reach the Google servers ("Request timeout"). In the third test, "sendto: No route to host" means that the network device knows where the Google servers are, but something along the digital pathway is broken.
+
+**Mac Users: How to run the "ping" command:**
+
+On a Mac, you typically run ping from the terminal command line. To start the terminal, click the OS X Spotlight magnifying glass icon in the upper right of the desktop:
+
+<img src="https://media.licdn.com/mpr/mpr/shrinknp_400_400/AAEAAQAAAAAAAAjWAAAAJDAzNzAyMTcwLTZlMzUtNGQ2ZC1hZGQ2LTc5NDA2NjA5ODJiYw.png" alt="Mac Spotlight">
+ 
+When the search window appears, type "terminal", highlight "Terminal – Utilities", and double-click (or hit return):
+<img src="https://media.licdn.com/mpr/mpr/shrinknp_800_800/AAEAAQAAAAAAAAjVAAAAJDZiNmY1MzQyLTYxZTUtNDA2My05ZTRhLTY3YjgwMDAzMjFmMA.png" alt="Mac Terminal Launch">
+
+That will launch the terminal command window, and you can enter the "ping Google.com" command (without quotes) shown in my examples:
+<img src="https://media.licdn.com/mpr/mpr/shrinknp_800_800/AAEAAQAAAAAAAAeIAAAAJDg2ZDhhYmQ4LTEwMTUtNGU4ZC1iYmJjLWE1MzMxMmY4MjEwYg.png" alt="Mac Command Line">
+ 
+**Important Mac Tip** : The ping command will run forever if you don't tell it to stop. To do that, press the control key (lower right on keyboard) and the “c” key. That will interrupt the test with a Control-C (^C ) and give back command line control. For Windows user, the command will stop by itself after a few iterations.
```

Comment:
@AndrewMichalik **Just a mini note:** Use the inline code i.e. **\`** (_backtick_) to surround the keyboard shortcut so that they appear like this :point_right: **[`Cmd` + `C`]**